src/vs/workbench/services/configuration/node/configuration.ts

```diff
@@ -416,28 +407,29 @@ export class WorkspaceServiceImpl extends WorkspaceService {
 			});
 	}
 
-	private onWorkspaceChange(configPath: URI): TPromise<void> {
-		let workspaceName = this.workspace.name;
-		this.workspaceConfigPath = configPath;
+	// TODO@Sandeep use again once we can change workspace without window reload
```

Comment:
@sandy081 fyi I commented out this code because we no longer have a workspace-change without window reload. I left the code in however because once we can support a workspace change without window reload, this could be used again possibly.